apps/web-giddh/src/app/contact/contact.component.html

```diff
@@ -93,7 +93,7 @@
                                 </sh-select>
                             </div>
                         </div>
-                        <div class="pull-left header-left">
+                        <div class="pull-left header-left" [ngClass]="{'on-company': currentCompanyBranches?.length <= 2 }">
```

Comment:
Please check for company with more than 2 branches and user is switched to company mode

Reply:
<!-- **On company** class will show only if we are on company mode and branches length will be less than equal 2 -->